### PR TITLE
Update commands

### PR DESCRIPTION
**Pull request by junrong98**
Shows consistency in commands between people and tasks

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditTaskCommand.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
             + PREFIX_TAG + "Homework";
 
     public static final String MESSAGE_EDIT_TASK_SUCCESS = "Updated Task: %1$s";
-    public static final String MESSAGE_NOT_EDITED = "At least one field to update must be provided.";
+    public static final String MESSAGE_NOT_EDITED = "A field has to be edited at least.";
```
Comment (junjunjieOng) [inline src/main/java/seedu/address/logic/commands/EditTaskCommand.java:45]: I think the statement, "At least one field to edit must be provided.", may sound better?🤔

Comment (junjunjieOng) [reply src/main/java/seedu/address/logic/commands/EditTaskCommand.java:45]: Just my opinion base on the EditCommand. 😆

File: docs/UserGuide.md [hunk 2]
```diff
@@ -283,8 +283,8 @@ Format: `filter KEYWORD [MORE_KEYWORDS]`
   e.g. `Apple Pear` will return `Eat apple`, `Buy pear`
 
 Examples:
-* `filter apple` returns `Buy apple` and `Make apple juice`
-* `filter orange pear` returns `Buy orange`, `Buy pear`<br>
+* `findt apple` returns `Buy apple` and `Make apple juice`
+* `findt orange pear` returns `Buy orange`, `Buy pear`<br>
```
Comment (brian16600) [inline docs/UserGuide.md:287]: I think the Command Summary at the bottom should be updated to? For standardisation

**Review by brian16600 — COMMENTED** [review]
Looks good to me. Just the UG needs some changing at the Command Summary, then I think you should be good to merge?

File: docs/UserGuide.md (hunk 2 above)
Comment (junrong98) [reply docs/UserGuide.md:287]: Gotcha

**Pull request merged**